src/UI/Areas/Identity/Pages/Account/Manage/DeletePersonalData.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@page
+@model DeletePersonalDataModel
+@{
+    ViewData["Title"] = "Delete Personal Data";
+    ViewData["ActivePage"] = ManageNavPages.DeletePersonalData;
+}
+
+<h4>@ViewData["Title"]</h4>
+
+<div class="alert alert-warning" role="alert">
+    <p>
+        <span class="glyphicon glyphicon-warning-sign"></span>
+        <strong>This action will delete your user.</strong>
+    </p>
+    <p>
+        This is unrecoverable.
+    </p>
+</div>
+
+<div>
+    <form method="post" class="form-group">
+        <button class="btn btn-danger" type="submit">Delete user and personal data</button>
+    </form>
+</div>
+
+@section Scripts {
+    @await Html.PartialAsync("_ValidationScriptsPartial")
```

Comment:
`<partial name="_ValidationScriptsPartial" />`

Reply:
We generally prefer tag helpers to HtmlHelpers. It's cleaner, more idiomatic code.